ios/FluentUI.Demo/FluentUI.Demo/Demos/ListDemoController.swift

```diff
@@ -11,90 +11,46 @@ class ListDemoController: DemoController {
         super.viewDidLoad()
 
         let sections = TableViewCellSampleData.sections
-        var section: TableViewCellSampleData.Section
         var cell: TableViewCellSampleData.Item
-        var indexPath = IndexPath(row: 0, section: 0)
         var showsLabelAccessoryView: Bool
 
         let list: MSFList = MSFList()
         var listCell: MSFListCellState
 
-        let samplePersonas: [PersonaData] = [
-            PersonaData(name: "Kat Larrson", email: "kat.larrson@contoso.com", subtitle: "Designer", image: UIImage(named: "avatar_kat_larsson"), color: Colors.Palette.cyanBlue10.color),
-            PersonaData(name: "Kristin Patterson", email: "kristin.patterson@contoso.com", subtitle: "Software Engineer", color: Colors.Palette.red10.color),
-            PersonaData(name: "Ashley McCarthy", image: UIImage(named: "avatar_ashley_mccarthy"), color: Colors.Palette.magenta20.color),
-            PersonaData(name: "Allan Munger", email: "allan.munger@contoso.com", subtitle: "Designer", image: UIImage(named: "avatar_allan_munger"), color: Colors.Palette.green10.color),
-            PersonaData(name: "Amanda Brady", subtitle: "Program Manager", image: UIImage(named: "avatar_amanda_brady"), color: Colors.Palette.magentaPink10.color)
+        //PersonaDataNode creation
+        let personaDataNodes: [PersonaDataNode] = [
+            PersonaDataNode(personaData: samplePersonas[0],
+                            children: [ PersonaDataNode(personaData: samplePersonas[1],
+                                                        children: [ PersonaDataNode(personaData: samplePersonas[2],
+                                                                                    children: [],
```

Comment:
You can omit the parameter if you're not passing any value different from the default.